upsetplot/plotting.py

```diff
@@ -269,9 +269,9 @@ class UpSet:
         The totals plot should be large enough to fit this many matrix
         elements.
     show_counts : bool or str, default=False
-        Whether to label the intersection size bars with the cardinality
-        of the intersection. When a string, this formats the number.
-        For example, '%d' is equivalent to True.
+        Whether to label the intersection size bars with the cardinality or
```

Comment:
I don't think I want to remove the ability to provide a format string for the raw counts.

So my docstring might be:
```
        Whether to label the intersection size bars with the cardinality
        of the intersection. If "percent", the value will be shown as 
        a percentage of all samples (to one decimal place). Otherwise,
        the string will be used as a format string for the number.
        True and '%d' are equivalent.
```

But you can tell me if I'm crazy and that's throwing too much functionality into one parameter. In which case, I'd rather avoid breaking backwards compat at this stage, and revert to having a separate show_percent param.

Reply:
Perhaps it's better to separate the parameters.  `show_counts` can behave as it did previously, accepting either a bool or formatting string and then I can add a bool `counts_as_percent` (or something like that) which converts the counts to percentages.

I'd propose, if one is not set by `show_counts`, the default formatting string should continue to be `%d` if `counts_as_percent` is False and `%.1f%%` if True.  This way, users can change the formatting string for both percentages and raw counts.

Let me know what you think and I can make the appropriate changes.